### ISSUE TITLE
Support for local storage emulator

### ISSUE THREAD

**Issue by vbeffa** [davidgraeff/firestore-db-and-auth-rs#25]
**Is your feature request related to a problem? Please describe.**
Good development practice is to use local database in local environment. Postgres and Firestore running locally is ideal.

**Describe the solution you'd like**
Ability to configure the credentials to point to the local firestore emulator.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by davidgraeff** [comment]
I agree. I haven't used a local firestore emulator yet, because it is so easy to just setup a -dev mirror / have a developer variant of an existing firestore. I'm open to PR to add this functionality.
